x/auth/tx/builder.go

```diff
@@ -63,8 +66,8 @@ func newBuilder(cdc codec.Codec) *wrapper {
 	}
 }
 
-func (w *wrapper) Size() int {
-	panic("not yet implemented")
+func (w *wrapper) Size() int64 {
+	return w.txSize
```

Comment:
This returns the correct value when decoding a tx.

However, wrapper is also used for constructing a tx. There are a lot of `Set*` methods. Each time a Set* is called, this value should be reset.

`Size` is only on the mempool.Tx interface though, which users shouldn't use when dealing with TxBuilders. Still, it could be confusing.

What I propose:
- set `w.txSize = 0` whenever we call a setter
- Add godocs on this method saying it shouldn't be used/will return 0 when called when building a tx

Reply:
Very good points!